content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1255,6 +1255,54 @@ const users = await prisma.user.createMany({
 })
 ```
 
+### `createManyAndReturn()`
+
+`createManyAndReturn` creates multiple records and returns the resulting objects.
+
+:::info
+
+`createManyAndReturn()` is only available in Prisma ORM version 5.14.0 and up.
+`createManyAndReturn()` is only available for PostgreSQL, CockroachDB, and SQLite.
+
+:::
+
+#### Options
+
+| Name              | Type                              | Required | Description                                                                                                                                                                                                                                             |
+| ----------------- | --------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data`            | `Enumerable<UserCreateManyInput>` | **Yes**  | Wraps all the model fields in a type so that they can be provided when creating new records. Fields that are marked as optional or have default values in the datamodel are optional.                                    |
+| [`select`](#select)            | `XOR<UserSelect, null>`  | No       | [Specifies which properties to include](/orm/prisma-client/queries/select-fields) on the returned objects.                                                                                 |
+| [`omit`](#omit-preview)        | `XOR<UserOmit, null>`    | No       | Specifies which properties to exclude on the returned objects. In [Preview](/orm/more/releases#preview) since 5.13.0. Mutually exclusive with `select`.                                            |
+| [`include`](#include)          | `XOR<UserInclude, null>` | No       | [Specifies which relations should be eagerly loaded](/orm/prisma-client/queries/relation-queries) on the returned objects.                                                                                 |
+| `skipDuplicates?` | `boolean`                         | No       | Do not insert records with unique fields or ID fields that already exist. Only supported by databases that support [`ON CONFLICT DO NOTHING`](https://www.postgresql.org/docs/9.5/sql-insert.html#SQL-ON-CONFLICT). This excludes MongoDB and SQLServer |
```

Comment:
I'm not sure if we have a test for `skipDuplicates` with `createManyAndReturn` @Weakky do you know?

Reply:
We do have a test

Reply:
https://github.com/prisma/prisma-engines/blob/54465718776ea99c622bd1c060a5fa445952886d/query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/top_level_mutations/create_many_and_return.rs#L214-L228